mm2src/coins/hd_wallet/account_ops.rs

```diff
@@ -14,6 +14,23 @@ use mm2_err_handle::prelude::*;
 pub trait HDAccountOps {
     type HDAddress: HDAddressOps + Clone + Send;
 
+    /// Creates a new `HDAccountOps` instance.
+    ///
+    /// # Parameters
+    ///
+    /// * `account_id`: The index of the account.
+    /// * `account_extended_pubkey`: The extended public key associated with this account.
+    /// * `account_derivation_path`: The derivation path from the master key to this account.
+    ///
+    /// # Returns
+    ///
+    /// A new `HDAccountOps` instance.
```

Comment:
Could you write doc comments above fields types (structures, pub types, enum etc), instead of function arguments repetition?
If some field needs specific note, you can this info as an additional sentence after `Creates a new `HDAccountOps` instance.`.
see https://github.com/KomodoPlatform/komodo-defi-framework/pull/2049#discussion_r1489331792 , https://github.com/KomodoPlatform/komodo-defi-framework/pull/2014#discussion_r1487411900

please do this for other functions

Reply:
Thanks for the reminder, I guess this commit was before these comments https://github.com/KomodoPlatform/komodo-defi-framework/pull/2049#discussion_r1489331792 , https://github.com/KomodoPlatform/komodo-defi-framework/pull/2014#discussion_r1487411900

Reply:
Fixed here https://github.com/KomodoPlatform/komodo-defi-framework/pull/1979/commits/503aafbf99b03908a2f485ccfaf66880cb4a6474